src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs

```diff
@@ -95,6 +95,14 @@
                 </div>
             </div>
 
+            <div id="x-pushbullet" class="form-group" style="display: none">
+                <label class="col-sm-3 control-label">Select device ID</label>
+                <div class="col-sm-5 form-inline">
+                  <select id="x-pushbullet-deviceIds" class="form-control"></select>
+                  <button class="btn">Populate list</button>
```

Comment:
Personally I think the button should be something like an icon with a refresh button, maybe even grouped with the input so they share the one edge (input-group) http://getbootstrap.com/components/#input-groups - not sure how this will look for a select though.